docs/components/tools.md

```diff
@@ -826,7 +832,7 @@ Flags:
       --rewrite.to-relabel-config-file=<file-path>
                                 Path to YAML file that contains relabel configs
                                 that will be applied to blocks
-      --tmp.dir="/tmp/thanos-rewrite"
+      --tmp.dir="/var/folders/qm/v1324vx15ysgw2nl4_p47h7c0000gn/T/thanos-rewrite"
```

Comment:
that seems weird

Reply:
Ups :D

Reply:
This means you are a macOS user, @nicolastakashi. Safe to remove this change.